src/ucp/tag/tag_match.inl

```diff
@@ -25,6 +25,29 @@
 #define UCP_TAG_MATCH_HASH_SIZE     1021
 
 
+#define ucp_tag_match_request_get(_worker, _param, _req, _failed) \
```

Comment:
plz rename these macros and move them to request.inl, otherwise we will have to do it in next PRs anyway

Reply:
can change in the next PR though